.github/workflows/nightly.yml

```
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
```

These should also be updated:

```suggestion
      - uses: actions/checkout@v4
      - uses: actions/setup-go@v5
```